docs/en/SUMMARY.md

```diff
@@ -76,6 +76,7 @@
   * [LED strip (legacy)](leds_old.md)
   * [Contribution Guidelines](contributing.md)
 * Clever-based projects
+  * [Drone show](clever-show.md)
```

Comment:
All the rest article filenames are with underscores.

Reply:
This is the name of the software: "clever-show"